### ISSUE TITLE
Firefox link is broken

### ISSUE THREAD

**Issue by Pajn** [apollographql/apollo-client-devtools#114]
The firefox addon link gives me a 404 and I can not find the extension on the Firefox add on site manually.

**Comment by vaneek** [comment]
@jbaxleyiii is Firefox addon supported or is development and testing occurring on the master branch?

**Comment by iamtchelo** [comment]
Still broken! :/

**Comment by eschaefer** [comment]
Until this is resolved, you can of course build the extension locally from this repo and install it on Firefox

```
git clone https://github.com/apollographql/apollo-client-devtools.git
npm i
npm run build
```
Open "about:debugging" in Firefox, click "Load Temporary Add-on" and select the `manifest.json` in `shells/webextension`.

Refresh the page you need to debug, and voila: 
<img width="1367" alt="screen shot 2018-05-14 at 14 44 41" src="https://user-images.githubusercontent.com/547148/39998365-28ccadc2-5786-11e8-8e45-53cf9f72570f.png">

**Comment by RobertB4** [comment]
Still no update? May as well switch to chrome until the firefox dev tools are working again.

**Comment by rkstar** [comment]
i've just gone back to firefox and i'm enjoying the experience.... really need this extension tho.

**Comment by joejordan** [comment]
So is Firefox supported or not? This issue has been open for 8 months(!) and the readme is still pointing to a deleted add-on page (https://addons.mozilla.org/en-US/firefox/addon/apollo-developer-tools/)

**Comment by hwillson** [comment]
Please see https://github.com/apollographql/apollo-client-devtools/issues/73#issuecomment-444497422. In a nutshell - devtools supports FF, but has been pulled by FF. We're working on a fix, and will track all progress in #73. Thanks!
